pkg/controller/secrets/secrets.go

```diff
@@ -65,22 +65,22 @@ func GenerateUserData(privateKey []byte) (*core.Secret, error) {
 			$containerLogsPort = "10250"
 			New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
 			Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
```

Comment:
We were installing this package provider so that we could install `OpenSSHUtils` and we should be able to remove this.